packages/editor-ui/src/components/TagsDropdown.vue

```diff
@@ -253,8 +253,7 @@ export default defineComponent({
 .tags-container {
 	$--max-input-height: 60px;
 
-	:deep(.el-select) {
-		.el-select__tags {
+		.el-tag {
```

Comment:
This now has unmatched curly brackets:
![image](https://github.com/n8n-io/n8n/assets/2598782/8318c1aa-054d-4764-aa1f-22236e87dde6)

Reply:
Ops, I deleted one line too many before committing. Thanks!